code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -68,6 +68,20 @@
 	skirts["jumper dress"] = /obj/item/clothing/under/skirt/jumper_highcut
 	gear_tweaks += new /datum/gear_tweak/path(skirts)
 
+/datum/gear/uniform/suitskirt
+	display_name = "Skirtsuit (recolourable)"
+	description = "A recolourable skirtsuit"
+	path = /obj/item/clothing/under/suit_jacket/skirtsuit/color
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+
+/datum/gear/uniform/command_suitskirt
+	display_name = "Command Skirtsuit"
+	description = "A Skirtsuit for Women in a Command Role"
```

Comment:
Personally, I

1. Wouldn't specify gender
2. Wouldn't capitalize each of these words. Maybe put a period at the end.

Reply:
Good call. 
For a variety of reasons.
Altered in my next update.
![grafik](https://user-images.githubusercontent.com/30761807/153253651-1f92ccd7-5b25-4b3d-808b-e5ad5ebda75b.png)